src/hotspot/share/prims/whitebox.cpp

```diff
@@ -74,6 +74,7 @@
 #include "runtime/javaCalls.hpp"
 #include "runtime/jniHandles.inline.hpp"
 #include "runtime/os.hpp"
+#include "runtime/safepoint.hpp"
```

Comment:
I don't think you need this include change anymore.

Reply:
Fixed